api/s2n.h

```diff
@@ -1882,6 +1882,18 @@ S2N_API extern uint64_t s2n_connection_get_delay(struct s2n_connection *conn);
  */
 S2N_API extern int s2n_connection_set_cipher_preferences(struct s2n_connection *conn, const char *version);
 
+/**
+ * Signals the connection to do a key_update at the next possible opportunity.
+ * 
```

Comment:
You imply it, but I think you explicitly need to call out that this method doesn't actually send the key update message, and that the application will need to call s2n_send to actually send it.